-1 
Tested at: 11992f70276aefe599cc8377c6d5b9999f8af9b0
 I found an error when building: 
 
 <pre>gmake[1]: *** [CompilePython] Error 1
gmake[1]: Target `PostBuild' not remade because of errors.
gmake[1]: Leaving directory `/build/mesos/slaves/20150418-223037-3834547840-5050-6-S28/frameworks/20150418-223037-3834547840-5050-6-2086/executors/mesos-jenkins-779adc29-eee6-4ebf-b8c9-84273251aedc/runs/39e524c5-5865-42ed-ad16-bc759a02abca/jenkins/workspace/ib-any-integration-mesos/CMSSW_7_4_X_2015-05-12-2300'
gmake: *** [src] Error 2
gmake: Target `all' not remade because of errors.
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
</pre> 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-9068/61/summary.html

--- a/DQM/Integration/python/test/visualization-live_cfg.py
+++ b/DQM/Integration/python/test/visualization-live_cfg.py
@@ -42,7 +42,7 @@ m = re.search(r"\((\w+)\)", str(source.runNumber))
 runno = str(m.group(1))
 outDir= '/fff/BU0/output/EvD/run'+runno
 
-create output directory
+#create output directory
 try:
     os.mkdir(outDir)
 except:

